src/providers/ListContinuationProvider.ts

```
@@ -0,0 +1,189 @@
+            const newPosition = new vscode.Position(position.line, 0);
+            editor.selection = new vscode.Selection(newPosition, newPosition);
+
+            return true;
+        }
+
+        // Continue the list
+        let newLineContent = '';
+
+        switch (patternType) {
+            case 'checkbox':
+                const marker = matchedPattern[2];
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)